next.config.mjs

```diff
@@ -101,32 +98,20 @@ export default withSentryConfig(
     nextConfig,
   ),
   {
-    // For all available options, see:
-    // https://github.com/getsentry/sentry-webpack-plugin#options
-
-    // Suppresses source map uploading logs during build
     silent: true,
-    release: process.env.SENTRY_RELEASE,
-  },
-  {
-    // For all available options, see:
-    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
-
-    // Upload a larger set of source maps for prettier stack traces (increases build time)
+    release: { create: false },
+    unstable_sentryWebpackPluginOptions: {
+      bundleSizeOptimizations: {
+        excludeDebugStatements: true,
+        excludeReplayShadowDom: true,
+        excludeReplayIframe: true,
+      },
+      release: { create: false },
```

Comment:
also should pass `release: { create: false }` here, otherwise release created by @sentry/webpack-plugin
even i already passed it 1 level upper https://github.com/r34son/profile/pull/422/files#diff-18c049b08c4a0f5ab451c598aeb2c4848bb9d7877b51ca3e5effb94a225814d2R102

Reply:
I don't want to create release on every build locally. I create release separately in ci

Reply:
Found why it was created, because i was using `_experiments: { injectBuildInformation: true },`
here https://github.com/getsentry/sentry-javascript-bundler-plugins/blob/2.16.0/packages/bundler-plugin-core/src/options-mapping.ts#L24. I don't expect to create release if i pass `create: false`

Reply:
> I don't want to create release on every build locally. I create release separately in ci

I'd just disable the webpack plugin entirely in environments where you don't want to create a release: https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#disable-sentrywebpackplugin